zerver/lib/events.py

```diff
@@ -216,6 +216,10 @@ def fetch_initial_state_data(user_profile: UserProfile,
         state['realm_push_notifications_enabled'] = push_notifications_enabled()
         state['realm_upload_quota'] = realm.upload_quota_bytes()
         state['realm_plan_type'] = realm.plan_type
+        state['plan_includes_wide_organization_logo'] = realm.plan_type != Realm.LIMITED
+        state['upgrade_text_for_wide_organization_logo'] = "Available on all paid plans. Upgrade "
+        # This string has been directly hardcoded for now but later on we might want
```

Comment:
We can remove this comment, but for future, if we were to have a commont here, it would go above the line it's commenting rather than below.